nixnet/database/_signal.py

```diff
@@ -483,19 +468,19 @@ def mux_value(self):
     @property
     def mux_subfrm(self):
         # actually returns _subframe.SubFrame, but avoiding a circular import
-        # type: () -> typing.Any
+        # type: () -> _database_object.DatabaseObject
```

Comment:
Should the return type just be SubFrame? Ditto as above.

Reply:
I was able to get this to work using the `MYPY = False` technique.